### ISSUE TITLE
[1.0] Memcache provider not working

### ISSUE THREAD

**Issue by izziaraffaele** [laravel-doctrine/orm#98]
I'm getting this error.

```
BindingResolutionException in Container.php line 823:
Unresolvable dependency resolving [Parameter #0 [ <required> $memcached ]] in class Illuminate\Cache\MemcachedStore
```

**Comment by sisve** [comment]
In what way? Any particular use-case that fails, or are you unable to use it from config files? Any error messages or any way we can reproduce the issue?

**Comment by patrickbrouwers** [comment]
Fixed by 1.0.10
